int32 	speech_cue_code		# Code for the type of speech interaction: 	1 - Ask one person to raise the cup
				#						2 - Ask one person to bring the cup closer to the robot
				#						3 - Ask one person to strighten up the cup
				#						4 - Ask multiple people to raise their cups
				#						5 - Ask multiple people to bring their cups closer to the robot
				#						6, 7, 8 - Correctly completed movements by the robot